quantum_systems/system.py

```diff
@@ -210,9 +210,7 @@ def u_t(self, current_time):
         if not self.has_two_body_time_evolution_operator:
             return u_0
 
-        return u_0 + sum(
-            op.u_t(current_time) for op in self._time_evolution_operator
-        )
+        return sum(op.u_t(current_time) for op in self._time_evolution_operator)
```

Comment:
What happened here? This is how it is now, isn't it?

Reply:
Sorry, I read the wrong branch, it should be
```python
        return u_0 + sum(
            op.u_t(current_time) for op in self._time_evolution_operator
        )
```

Reply:
Could you restore this function to what it was. Unless you mean that there is a reason to why it should be like you have made it. The extra `u_0` is supposed to be there.